ac.soton.coda.vhdl.tests.feature/feature.xml

```diff
@@ -1,22 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+    (c) Crown owned copyright (2017) (UK Ministry of Defence)
+   
+    All rights reserved. This program and the accompanying materials are 
+    made available under the terms of the Eclipse Public License v1.0 which
+    accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+         University of Southampton - Initial API and implementation
+ -->
 <feature
       id="ac.soton.coda.vhdl.tests.feature"
       label="%featureName"
-      version="0.0.1.qualifier"
+      version="0.0.5.qualifier"
       provider-name="%featureVendor"
-      plugin="ac.soton.coda.vhdl.branding">
+      plugin="ac.soton.coda.vhdl.branding"
+      license-feature="ac.soton.coda.licence"
+      license-feature-version="1.1.2.release">
```

Comment:
check licence version no. I still have 1.1.1.qualifier

Reply:
Oops, I think I have 1.1.2.release

Reply:
I think it is ok to stay with a qualifier for now (it doesn't auto build I think).. Or we could release it since it probably won't change. However, according to the commit history it looks like 1.1.1 not 1.1.2?

Reply:
I think we should make it 1.1.1.release. I cannot remember why it was 1.1.2.release for VHDL. I think the .release was because it is not auto build. I will put in a request soon for Utils.